### ISSUE TITLE
ci: install latest bbi automatically?

### ISSUE THREAD

**Issue by kyleam** [metrumresearchgroup/rbabylon#620]
It'd be nice if we didn't have to update the drone config for each release (e.g., gh-619).  We could update [install_bbi.sh](https://github.com/metrumresearchgroup/bbr/blob/df66feb801427ceea8704a27c5c1a7674e60782f/install_bbi.sh) to query https://api.github.com/repos/metrumresearchgroup/bbi/releases/latest, though we'd need handling for if the rate limit is hit (especially if we aren't using an authenticated user).

**Comment by seth127** [comment]
That makes sense to me. Could we still do [this skip](https://github.com/metrumresearchgroup/bbr/blob/df66feb801427ceea8704a27c5c1a7674e60782f/install_bbi.sh#L24) if we're querying `latest` instead of a version? I'm not sure what exactly that buys us, but I think it probably makes sense.

Also, I assume we'd still do the old frozen version for the "oldest" pipeline, as noted in gh-619?

**Comment by kyleam** [comment]
> Could we still do [this skip](https://github.com/metrumresearchgroup/bbr/blob/df66feb801427ceea8704a27c5c1a7674e60782f/install_bbi.sh#L24) if we're querying `latest` instead of a version? I'm not sure what exactly that buys us, but I think it probably makes sense.

Yes, we could still run that once we got the latest version for via the api.github.com query.

(In terms of what it buys us, the comment following that makes me think it doesn't save use from many installs, but I'd need to look through the drone config and probably read up on some settings to have a better guess.)

> Also, I assume we'd still do the old frozen version for the "oldest" pipeline, as noted in [gh-619](https://github.com/metrumresearchgroup/bbr/pull/619)?

Right.

**Comment by kyleam** [comment]
We've moved to GitHub Actions, but this still applies.  However, the core implementation would now be in the metrumresearchgroup/actions repo:

https://github.com/metrumresearchgroup/actions/blob/a286445062005feb83408d893816dff55592cdd6/setup-bbi/action.yaml#L5-L9

```
inputs:
  # TODO: At some point, it'd be nice if the default was to
  # automatically detect new versions (bbr 620).
  version:
    description: Version of bbi to install
```
